### ISSUE TITLE
IrcUser.Quit event

### ISSUE THREAD

**Issue by hypodyne** [IrcDotNet/IrcDotNet#67]
Hi All
The IrcUser.Quit event is not always being called leaving me with ghost users in my nicklist. I see in the source that its not always sent. How do you handle the user quits so its accurate?
thanks.

```
       /// <summary>
        ///     Occurs when the user has quit the network. This may not always be sent.
        /// </summary>
        public event EventHandler<IrcCommentEventArgs> Quit;
```

**Comment by txdv** [comment]
You mean that everything is fine with IrcUser.Quit in IrcDotNet?

I mean, you have the source code, why don't you just look it up?

**Comment by hypodyne** [comment]
Hi txdv
I did look it up and found that quit events are not always sent. Why? I don't know.

**Comment by txdv** [comment]
Sometimes people part from channels? Did you consider that ?

**Comment by hypodyne** [comment]
I am subscribing to both quit and part.  I run IrcDotNet side by side with IceChat, Sometimes people leave from Icechat but are still in IrcDotNet. The quit event is just not being called sometimes as the source code snippet suggests.

**Comment by hypodyne** [comment]
The strange thing is that the Channel.Users.Count is correct the next time someone leaves, joins or quits. But the quit event is definitely not being thrown every time. About 30% are thrown okay.

I tried to trace this via the source. When I got to `protected internal void ProcessMessageQuit(IrcMessage message)` Visual Studio could not find any further references.

I am using it as a unity3d plugin, so I can't attach the debugger while running.
